### ISSUE TITLE
C3M1_A roof issue

### ISSUE THREAD

**Issue by hawkwind3** [Realm667/WolfenDoom#1507]
The sides of the roof are not being rendered. Goes back to at least BoA 3.1.
I tried to fix with minimal success.
Please see pic .
![roof bug](https://github.com/user-attachments/assets/c181d3e5-847d-4948-af9a-28a09e44c53a)

**Comment by Tormentor667** [comment]
You aren't supposed to fly here anyway ;)

**Comment by hawkwind3** [comment]
Of course. Just did that to make it more noticeable. Can see from the ground. Walk around the building. Especially noticed at the peaks of the roof.
